_component_design/modal.md

```diff
@@ -127,23 +173,21 @@ Refer to the [Loading pattern]({{site.url}}/components/loading-indicators.html)
 
 {% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
 
-### Use Case: Confirmation
-
-Confirmation modals interrupt the user before they complete a given workflow and require a single decision from the user.
+## Confirmation modals
 
-The modal is triggered by the user submitting information or initiating a process that will have significant or destructive impact.
+A confirmation modal interrupts a user and requires a decision before the user can continue working. The modal is triggered when the user initiates a process or submits information that can have a significant or destructive impact.
 
-**Confirmation scenarios include the following actions:**
+**Confirmation scenarios include actions that result in:**
 
-- Action that triggers deletion or overwriting of data
+- Deleting or overwriting data
 - Loss of unsaved data
-- Changes that will impact billing
+- Changes that impact billing
 
-**All confirmation modals include the following elements:**
+**Confirmation modals include:**
```

Comment:
Per style (http://rackerlabs.github.io/docs-rackspace/style-guide/style/lists.html#introductory-text), an introduction for a list should be a sentence versus a fragment.